src/Exception/ExceptionMapper.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: kasper
+ * Date: 10/01/2018
+ * Time: 14.34
+ */
+
+namespace Drupal\apsis_mail\Exception;
+
+
+use GuzzleHttp\Exception\RequestException;
+
+/**
+ * Mapper for transforming generic HTTP exceptions to specific ApsisExceptions.
+ *
+ * Officially APSIS uses a number of different types of errors. This set is larger than the number of used HTTP status
```

Comment:
"In general, all lines of code should not be longer than 80 characters."

Reply:
:+1: